key-script

```diff
@@ -22,16 +22,19 @@ message()
 
 check_yubikey_present="$(ykinfo -q -"$YUBIKEY_LUKS_SLOT")"
 
-if [ -z "$cryptkeyscript" ]; then
-    if [ -x /bin/plymouth ] && plymouth --ping; then
-        cryptkeyscript="plymouth ask-for-password --prompt"
-    else
-        cryptkeyscript="/lib/cryptsetup/askpass"
-    fi
+if [ -z "$YKFDE_CHALLENGE" ] || [ "$check_yubikey_present" -ne "1" ] ; then
```

Comment:
Why the `"$check_yubikey_present" -ne "1"` is needed here?

Reply:
Without it, there is no way for you to get it to prompt for the (recovery) password interactively (by removing the Yubikey). If it doesn't check for it being present, first, then it just loops infinitely (or until the LUKS script has been ran too many times and it forces a restart)---attempting to get the challenge response from an unattached Yubikey.